setup.py

```diff
@@ -9,17 +9,5 @@
             'mizard=mizar.daemon.app:main'
         ]
     },
-    install_requires=[
-        'PyYAML',
-        'kopf',
-        'netaddr',
-        'ipaddress',
-        'pyroute2',
-        'rpyc',
-        'kubernetes==11.0.0',
-        'luigi==2.8.12',
-        'grpcio',
-        'protobuf',
-        'fs'
-    ]
+    install_requires=[]
```

Comment:
I don't think we should remove these. If they are already installed in the image, that's fine. But we need to keep the dependencies in the setup script anyway.

Reply:
Ok I'll re-add them